templates/webhook.service.erb

```diff
@@ -3,19 +3,16 @@ Description=R10K Webhook Service
 After=syslog.target network.target
 
 [Service]
-Type=forking
+Type=simple
```

Comment:
To change this, wouldn't you also have to change the process such that it doesn't daemonise and create the pidfile?

Reply:
Oh, ignore that, you have.  Still not convinced we should mess around too much just because docker has an issue.

Reply:
Actually, not just docker, it affects regular installations as well
That RedHat article I linked above recommending not using Type=forking as a legacy method.